static/app/views/onboarding/targetedOnboarding/components/createProjectsFooter.tsx

```diff
@@ -79,7 +82,30 @@ export default function CreateProjectsFooter({
         organization,
       });
       clearIndicators();
-      onComplete();
+      let isMobile: boolean = window.innerWidth < 800;
+      if ((navigator as any).userAgentData) {
+        isMobile = (navigator as any).userAgentData.mobile;
+      }
+      if (
+        isMobile &&
+        organization.experiments.TargetedOnboardingMobileRedirectExperiment === 'hide'
+      ) {
+        if (shouldSendEmail) {
+          persistedOnboardingState &&
+            api.requestPromise(
```

Comment:
@Neo-Zhixing maybe we should wait until this succeeds before sending the user somewhere else